ethereum/balancer/view_vebal_votes.sql

```diff
@@ -76,7 +76,6 @@ WITH calendar AS (
         ON v.round_id <= r.round_id
         AND r.round_id < v.next_round
         AND r.end_timestamp <= v.unlocked_at
-        AND v.weight > 0
```

Comment:
i'm unable to drop this view as 'dune_user_generated.vw_tianqi_gauge_reward' depends on it -- is this your user generated object? would you like to back it up, then drop, then allow this rebuild?

Reply:
it's not mine but I know the user who maintains it. I don't think we need to back it up - I'm just fixing some issues in both views here and didn't change anything else.